business_rules/engine.py

```diff
@@ -107,5 +108,7 @@ def fallback(*args, **kwargs):
                                  .format(method_name, defined_actions.__class__.__name__))
 
         params = action.get('params') or {}
+        if returned_values:
+            params = {**params, **returned_values}
```

Comment:
i'm unsure of precedence here (tbh in any direction) - previous action results overwrite action params or vice versa